lib/CGameState.h

```diff
@@ -81,6 +81,34 @@ struct DLL_LINKAGE SThievesGuildInfo
 
 };
 
+struct DLL_LINKAGE RumorState
+{
+	enum ERumorType : ui8
+	{
+		TYPE_NONE = 0, TYPE_RAND, TYPE_SPECIAL, TYPE_MAP
+	};
+
+	enum ERumorTypeSpecial : ui8
+	{
+		RUMOR_OBELISKS = 208,
+		RUMOR_ARTIFACTS = 209,
+		RUMOR_ARMY = 210,
+		RUMOR_INCOME = 211,
+		RUMOR_GRAIL = 212
+	};
+
+	ERumorType type;
+	std::map<ERumorType, std::pair<int, int>> last;
+
+	RumorState(){type = TYPE_NONE; last = {};};

```

Comment:
Before I can't test it because I have't got any time, so now when I test it, this one provides me an error: operator =' is ambiguous